File: bacalhao-a-bras-java/src/main/java/org/jesperancinha/portuguese/recipes/bras/TaskSimulator.java
package org.jesperancinha.portuguese.recipes.bras;

import org.joda.time.DateTime;

public class TaskSimulator {

    public static void waitMilliseconds(String name, long millisecondsTimes10){
        for (int i = 0; i < millisecondsTimes10; i++) {
            try {
                Thread.sleep(10);
            } catch (InterruptedException e) {
                e.printStackTrace();
            }
            System.out.println(String.format("%s, 1, %s", name, DateTime.now().toString()));
        }
    }
}

File: peixinhos-da-horta-parallel-scala/src/main/scala/org/jesperancinha/portuguese/recipes/peixinhos/TaskSimulator.scala
package org.jesperancinha.portuguese.recipes.peixinhos

import org.joda.time.DateTime

object TaskSimulator {

  def waitMilliseconds(name: String, millisecondsTimes10: Long): Unit = {
    var i = 0
    while ( {
      i < millisecondsTimes10
    }) {
      try Thread.sleep(10)
      catch {
        case e: InterruptedException =>
          e.printStackTrace()
      }
      System.out.println(String.format("%s, 1, %s", name, DateTime.now.toString))

      i += 1
    }
  }
}
